docker/anaconda3/Dockerfile

```diff
@@ -1,17 +1,26 @@
-# DOCKER-VERSION 1.11.1
-#
 # docker build --tag alleninstitute/allensdk:anaconda3 .
 # docker run -it alleninstitute/allensdk:anaconda3 /bin/bash
 #
 FROM continuumio/miniconda3
```

Comment:
this is not a static tag base image. Probably ok, but, could pin to 4.8.2

Reply:
Done!